.github/workflows/buildkit.yml

```diff
@@ -69,13 +69,17 @@ jobs:
       -
         name: BuildKit ref
         run: |
-          echo "BUILDKIT_REF=$(./hack/buildkit-ref)" >> $GITHUB_ENV
+          # FIXME(tonistiigi) test suite needs patch moby/buildkit#3567
+          # echo "BUILDKIT_REPO=moby/buildkit" >> $GITHUB_ENV
+          # echo "BUILDKIT_REF=$(./hack/buildkit-ref)" >> $GITHUB_ENV
+          echo "BUILDKIT_REPO=tonistiigi/buildkit" >> $GITHUB_ENV
+          echo "BUILDKIT_REF=db67180a1a439efb1547ecf5decd4003ec8f621b" >> $GITHUB_ENV
```

Comment:
Note to self; we can remove this once https://github.com/moby/buildkit/pull/3567 is merged and cherry-picked in the 0.11 branch